ui/src/Pages/Home/CartPage.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect } from "react";
+import { Container, Grid, Typography, Button, Box, Paper, Divider } from "@mui/material";
+import CartProducts from "../../components/Products/CartProducts";
+import {Link as RouterLink} from "react-router-dom";
+
+function CartPage() {
+  const subtotal = 5000;
+  const discount = 20;
+  const delivery = 20;
+  const tax = 20;
+  const total = subtotal - discount + delivery + tax;
```

Comment:
you could make a folder under `ui/src` for test data and import it into this file. Ask me or chatgpt if you don't get what I mean.

Reply:
You could do this for test value you hard code in, like the date and Standard/Express shipping prices you have below. This is how it will work anyways when we add the backend.